srcs/requirements/nginx/conf/phemsi-a

```diff
@@ -2,34 +2,24 @@ server {
 	listen 443 ssl;
 	listen [::]:443 ssl;
 
-    # This defines the server name and the server block that should be used for requests to the server
 	server_name DOMAIN_NAME www.DOMAIN_NAME;
-
-    # Our root directive names the root directory for requests to our server.
+	
 	root /var/www/phemsi-a/wordpress;
+	index index.php index.html index.htm;
+	
+	include snippets/self-signed.conf;
+	include snippets/ssl-params.conf;
 
-    # The index directive defines the files that will be used as indexes when processing requests to your server.
-    index index.php index.html;
-
-    # SSL Settings imports
-    include snippets/self-signed.conf;
-    include snippets/ssl-params.conf;
-
-    # In this location block, we’ll use a try_files directive to check for files that match individual URI requests. Instead of returning a 404 Not Found status as a default, however, we’ll pass control to WordPress’s index.php file with the request arguments.
 	location / {
```

Comment:
```suggestion
 # In this location block, we’ll use a try_files directive to check for files that match individual URI requests. Instead of returning a 404 Not Found status as a default
	location / {
```